cfglib.py

```diff
@@ -174,6 +174,16 @@
                           title='Config RabbitMQ for source cloud')
 
 src_rabbit_opts = [
+    cfg.StrOpt('db_user', default=None,
+               help='user for mysql'),
+    cfg.StrOpt('db_password', default=None,
+               help='password for mysql'),
+    cfg.StrOpt('db_host', default=None,
+               help='host of mysql'),
+    cfg.IntOpt('db_port', default=None,
+               help='port for mysql connection'),
+    cfg.StrOpt('db_connection', default=None,
+               help='driver for connection'),
```

Comment:
I might be missing something, but do we really need DB creds for rabbit?.. My understanding is that for rabbit we only need rabbit-specific creds, not mysql.